sca/src/test/resources/sca.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "variableNamingRule" : "^[a-z]+(?:[A-Z][a-z]*)*$",
+  "printLnArgumentNonExpressionRule" : true
+}
```

Comment:
Why should we enter the regex and not explicitly say camel case or snake case

Reply:
To make it more extensible. This way there is no need of creating new configurers for each naming convention.